### PR TITLE
Deprecated support for typing commands without k|kc|kubectl prefix targeting version 0.15

### PR DESCRIPTION
**Pull request by ezodude**
## Description

Changes proposed in this pull request:

- Adds required `kubectl` prefix to all examples.
E.g. changes documented commands from `@Botkube get po` to `@Botkube kubectl get po`.

The task was simplified after the latest updates to docs main.

## Related issue(s)

See also https://github.com/kubeshop/botkube/issues/802

### CODE REVIEW

File: docs/configuration/executor.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ The default configuration for Helm chart can be found in the [values.yaml](https
...
 
-- `@Botkube get po/botkube -n botkube` - collect `kubectl` executor bindings that include `botkube` or `*.` (all) Namespaces.
-- `@Botkube get po -A` - collect all `kubectl` executor bindings that include `*.` (all) Namespaces.
-- `@Botkube get po` - first, we resolve the execution Namespace. For that, we collect all enabled `kubectl` executor bindings and check the `defaultNamespace` property. If property is not define, we use the `default` Namespace. With resolved execution Namespace, we run the logic define in the first step.
+- `@Botkube kubectl get po/botkube -n botkube` - collect `kubectl` executor bindings that include `botkube` or `*.` (all) Namespaces.
+- `@Botkube kubectl get po -A` - collect all `kubectl` executor bindings that include `*.` (all) Namespaces.
+- `@Botkube kubectl get po` - first, we resolve the execution Namespace. For that, we collect all enabled `kubectl` executor bindings and check the `defaultNamespace` property. If property is not define, we use the `default` Namespace. With resolved execution Namespace, we run the logic define in the first step.
```
Comment (madebyrogal) [inline docs/configuration/executor.md:82]: From now on it will be required to have kubectl prefix?

Comment (ezodude) [reply docs/configuration/executor.md:82]: Yup, we're taking the deprecation notice to the next level, [see here](https://github.com/kubeshop/botkube/issues/802).

>In 0.15:
>
> - we change [the warning](https://botkube.io/docs/usage/#debugging-deployments-using-botkube) and say that k|kc|kubectl will be required in the 0.17 version
> - we replace all text examples and add required prefix e.g. change command from `@Botkube get po` to `@Botkube kubectl get po` in the Help message. 
>
>announce that in the release notes

Comment (madebyrogal) [reply docs/configuration/executor.md:82]: We have something similar in https://botkube.io/docs/usage/
Exactly this:
Screenshot 2022-10-18 at 10 08 47

WDYT to make it in the similar way?

Comment (ezodude) [reply docs/configuration/executor.md:82]: This has already been removed in @pkosiec latest doc re-organisation.

So no longer an issue 🙂 .. See https://botkube.io/docs/next/installation/

Comment (ezodude) [reply docs/configuration/executor.md:82]: Yup, I just found it and updated it below 👇 .

**Review by madebyrogal — APPROVED** [review]

**Pull request merged**